src/OpenTelemetry/Metrics/AggregatorStore.cs

```diff
@@ -255,15 +262,19 @@ private void UpdateLong(long value, ReadOnlySpan<KeyValuePair<string, object>> t
                 var index = this.FindMetricAggregatorsDefault(tags);
                 if (index < 0)
                 {
-                    // TODO: Measurement dropped due to MemoryPoint cap hit.
+                    if (Interlocked.CompareExchange(ref this.metricCapHitMessageLogged, 1, 0) == 0)
+                    {
+                        OpenTelemetrySdkEventSource.Log.MeasurementDropped(this.name, this.metricPointCapHitMessage, "Modify instrumentation to reduce the number of unique key/value pair combinations. Or use MeterProviderBuilder.SetMaxMetricPointsPerMetricStream to set higher limit.");
```

Comment:
I think there is a general problem - when should we log issue related to the measurement reporting? Considering the logging API is much slower.

Reply:
Maybe - instead of reporting these via logs, we can have some built-in metrics (e.g. dropped count).

Reply:
yes agree.
Will address it as future.